src/components/LoginComponent.tsx

```diff
@@ -3,7 +3,7 @@ import { Formik } from "formik";
 import { connect } from "react-redux";
 import { Button, Text, View, TextInput, StatusBar } from "react-native";
 import styles from "./Style";
-import { Redirect } from "../utilities/routing";
+import { Redirect } from "react-router";
```

Comment:
Same here, does this correctly preserve the react-native routing? That is the reason the routing is done in the utilities.